README.md

```diff
@@ -36,6 +36,7 @@ Provides distributed tracing for Play Framework using [Zipkin](https://zipkin.io
 2. Run zipkin-api-play24 project
 
   ```
+  $ git checkout 1.2.0 -- sample/zipkin-api-play24
```

Comment:
If we run this command in the same repository as above one for zipkin-api-play27, I guess zipkin-api-play27 won't work because it doesn't exist in 1.2.0 tag. It might be better to use separate repositories for Play 2.7 and 2.4 in this instruction if we drop old examples from master.

Reply:
It works because the directory `sample/zipkin-api-play24` will be checked out, even though it is no longer in the working tree. But no strong opinion regarding this.

Reply:
Aha, just checkout only `zipkin-api-play24` directory into `sample` directory without any changes to existing directories. I tried this procedure and it worked perfectly! 👍